### ISSUE TITLE
Can't access swagger page because of CSP

### ISSUE THREAD

**Issue by HRK44** [FoalTS/foal#969]
I followed the docs to create the swagger docs/page accessible at localhost/swagger on my app.

But when I try to access the page, I have this error in the browser :

> Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-qarl5JfyZV5piFIYvivIrdE71d2UL/zZYuxObwkSrLI='), or a nonce ('nonce-...') is required to enable inline execution.

I did a bit of research and it seems that I need to configure `helmet` (which I am using) to modify the CSP to  :

```javascript
    expressApp.use(helmet({
        contentSecurityPolicy: {
            directives: {
                ...helmet.contentSecurityPolicy.getDefaultDirectives(),
                'script-src': ['\'self\'', '\'unsafe-inline\''] //required for swagger docs
            }
        }
    }));
```

I don't think it's safe to add `unsafe-inline` to be able to view the swagger page. Would it be possible to somehow put the inline scripts into a .js file?

Resource : https://stackoverflow.com/a/64729487/7991096

**Comment by LoicPoullain** [comment]
What is your browser? What browser version do you use?

**Comment by HRK44** [comment]
@LoicPoullain Google Chrome Version 91.0.4472.114 (Official Build) (64-bit) on Ubuntu 20

**Comment by LoicPoullain** [comment]
I removed the inline script in the swagger page. The enhancement will be released with v2.6.

In the meantime, you might use this to disable the strict CSP on the swagger controller:
```typescript
import { SwaggerController } from '@foal/swagger';
import { Hook } from '@foal/core';
import { ProfileController } from './profile.controller';

@Hook(() => response => {
  response.setHeader('Content-Security-Policy', 'script-src \'self\' \'unsafe-inline\'')
})
export class OpenapiController extends SwaggerController {

  options = {
    controllerClass: ProfileController
  };

}

```
